src/controllers/AbstractController.ts

```diff
@@ -164,6 +164,26 @@ export default abstract class AbstractController<T extends AbstractService> {
 		return num;
 	}
 
+	protected parseQueryParamArrayOrThrow(
+		n: string,
+		errorMessage: string
+	): number[] {
+		const isValidParam = /[^,]+/.test(n);
+
+		if (!isValidParam) {
+			throw new BadRequest(errorMessage);
+		}
+
+		return n
+			.split(',')
+			.map((str) =>
+				this.parseNumberOrThrow(
+					str,
+					`Incorrect AssetId format: ${str} is not a U32.`
```

Comment:
Yea it should be fine. I decided to put the primitive type of the `assetId` in the error message for more accurate messaging instead of just saying `number`, I don't think an assetId above 2,147,483,647 will ever exist though. Do you think it would be better to replace the `U32` with number?

Reply:
I would just say "is not a positive integer" since that seems the "most" correct in this case.

Reply:
Fixed!